docs/sources/administration/service-accounts/add-service-account-token.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Add a token to a service account in Grafana'
+menuTitle: 'Add token for service account'
+aliases: [docs/sources/administration/service-accounts/add.md]
+description: 'This contains information about adding a token to a service account'
+weight: 60
+---
+
+# Add a service account token to a service account in Grafana
+
+You can add a token to service account in the service account tab of the organization.
+For more information about service accounts refer to [About service accounts in Grafana]({{< relref "./about-service-accounts.md">}}).
+
+For more information about creating a service account token via the API, refer to [HTTP API Create service account token]({{< relref "../../http_api/serviceaccount.md#create-service-account-tokens">}}).
+
+Before you begin:
+
+- Ensure you have added the feature toggle for service accounts `service-accounts`. Refer to [Enable service accounts]({{< relref "./enable-service-accounts.md#">}})
+- Ensure you have permission to create and edit service accounts, for more information refer to [About users and permissions]({{< relref "../manage-users-and-permissions/about-users-and-permissions.md#">}})
```

Comment:
Is there a specific role they should be granted? If so, link them directly to that section in the About topic.